python/certificates/disabled-cert-validation.yaml

```diff
@@ -0,0 +1,40 @@
+rules:
+    - id: disabled-cert-validation
+      patterns:
+        - pattern-either:
+            - pattern: urllib3.PoolManager(..., cert_reqs=$REQS, ...)
+            - pattern: urllib3.ProxyManager(..., cert_reqs=$REQS, ...)
+            - pattern: urllib3.HTTPSConnectionPool(..., cert_reqs=$REQS, ...)
+            - pattern: urllib3.connectionpool.HTTPSConnectionPool(..., cert_reqs=$REQS, ...)
+            - pattern: urllib3.connection_from_url(..., cert_reqs=$REQS, ...)
+            - pattern: urllib3.proxy_from_url(..., cert_reqs=$REQS, ...)
+            - pattern: $CONTEXT.wrap_socket(..., cert_reqs=$REQS, ...)
+            - pattern: ssl.wrap_socket(..., cert_reqs=$REQS, ...)
+        - pattern-where-python: |-
+            vars['$REQS'] in ['NONE', 'CERT_NONE', 'ssl.CERT_NONE', 'ssl.CERT_OPTIONAL', 'CERT_OPTIONAL',
```

Comment:
I don't think the `'NONE'` check is valid. I believe that's looking for `ssl.NONE`, which isn't an attribute:

```python
----> 1 ssl.NONE

AttributeError: module 'ssl' has no attribute 'NONE'
```

Reply:
It's *technically* not, but I left it in in case someone makes a variable named NONE and passes it in. Maybe a wrong decision.